### PR TITLE
RATEST-74:Failing Test Ignored AddFormTests

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RATEST-74
cc @k-joseph  @dkayiwa , kindly review thanks

### CODE REVIEW

**Comment by sherrif10** [comment]
Hey @k-joseph this is first failing from clickOnLast(visit_note); any more suggestion on this

**Comment by sherrif10** [comment]
@k-joseph  it seems we need to create this method clickOnLast  in uitest framework under page as clickOn was created https://github.com/openmrs/openmrs-contrib-uitestframework/blob/master/src/main/java/org/openmrs/uitestframework/page/Page.java#L196 , how do you see that

File: ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
     private static final By EXTENSION_FORM_LABEL = By.id("extensionForm.label");
     private static final By EXTENSION_FORM_ICON = By.id("extensionForm.icon");
     private static final By DELETE = By.cssSelector("i.icon-remove.delete-action");
+	public static final Object FORM_EXIST =  By.linkText("formEdit.form?formid=");
```
Comment (HerbertYiga) [inline ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java:18]: this can be on the same line like the other statements

**Comment by kdaud** [comment]
> @k-joseph it seems we need to create this method clickOnLast in uitest framework under page as clickOn was created https://github.com/openmrs/openmrs-contrib-uitestframework/blob/master/src/main/java/org/openmrs/uitestframework/page/Page.java#L196 , how do you see that

@sherrif10 do you suggest including this method on the master branch otherwise the method exists on the qa branch of the uitestframework. If its true  that qa branch travis is building the right dependent qa branch which contains the method then **I think there should be some thing to be done**. @k-joseph did you merge the pushed commit to skip that test for now

**Comment by sherrif10** [comment]
sure thanks for that catch

**Comment by kdaud** [comment]
@sherrif10 I recommend after pulling the latest changes as I mentioned above, ensure that your dedicated branch is identical to your local qa branch. On your dedicated branch under construction, do something like _git reset --hard qa_ and then build the branch again before commit

**Comment by sherrif10** [comment]
cc @k-joseph  , i think i was some where wrong with this method ,do you have some suggestions on this

**Pull request closed without merging**